google_drive/tests/__init__.py

```diff
@@ -0,0 +1 @@
+""" Put tests here """
```

Comment:
New pylint requires that there be a comment within this file, even though an empty version of the file is a common thing